### PR TITLE
Update changelog for 4.7.0.

### PR DESCRIPTION
**Pull request by hodgestar**
**Description**
Update changelog for 4.7.0.

**Related issues or PRs**
- See https://github.com/qutip/qutip/milestone/12?closed=1

**Changelog**
Update changelog for 4.7.0.

### CODE REVIEW

File: doc/changelog.rst [hunk 1]
```diff
@@ -4,7 +4,84 @@
...
+- Added transparency parameter to the add_point, add_vector and add_states methods in the Bloch and Bloch3d classes. (`#1837 <https://github.com/qutip/qutip/pull/1837>`_ by Xavier Spronken)
+- Support ``Path`` objects in ``qutip.fileio``. (`#1813 <https://github.com/qutip/qutip/pull/1813>`_ by Adrià Labay)
+- Improved the weighting in steadystate solver, so that the default weight matches the documented behaviour and the dense solver applies the weights in the same manner as the sparse solver. (`#1275 <https://github.com/qutip/qutip/pull/1275>`_ and `#1802 <https://github.com/qutip/qutip/pull/1802>`_ by NS2 Group at LPS and Simon Cross)
+- Added a ``color_style`` option to the ``hinton`` plotting function. (`#1595 <https://github.com/qutip/qutip/pull/1595>`_ by Cassandra Granade)
+- Improved the scaling of ``floquet_master_equation_rates`` and ``floquet_master_equation_tensor`` and fixed transposition and basis change errors in ``floquet_master_equation_tensor`` and ``floquet_markov_mesolve``. (`#1284 <https://github.com/qutip/qutip/pull/1284>`_ by Camille Le Calonnec, Jake Lishman and Eric Giguère)
```
Comment (Ericgig) [inline doc/changelog.rst:33]:
```suggestion
- Improved the scaling of ``floquet_master_equation_rates`` and ``floquet_master_equation_tensor`` and fixed transposition and basis change errors in ``floquet_master_equation_tensor`` and ``floquet_markov_mesolve``. (`#1248 <https://github.com/qutip/qutip/pull/1248>`_ by Camille Le Calonnec, Jake Lishman and Eric Giguère)
```

**Review by Ericgig — COMMENTED** [review]
PRs that slipped in without the milestone: #1840,  #1828

File: doc/changelog.rst [hunk 2]
```diff
@@ -4,7 +4,83 @@
...
+- The signatures of rand_ket and rand_ket_haar were changed to allow N (the size of the random ket) to be determined automatically when dims are specified. (`#1509 <https://github.com/qutip/qutip/pull/1509>`_ by Purva Thakre)
+
+Bug Fixes
+---------
+- Fix circuit index used when plotting circuits with non-reversed states. (`#1847 <https://github.com/qutip/qutip/pull/1847>`_ by Christian Staufenbiel)
+- Changed implementation of ``qutip.orbital`` to use ``scipy.special.spy_harm`` to remove bugs in angle interpretation. (`#1844 <https://github.com/qutip/qutip/pull/1844>`_ by Christian Staufenbiel)
+- Fixed ``QobjEvo.tidyup`` to use ``settings.auto_tidyup_atol`` when removing small elements in sparse matrices. (`#1832 <https://github.com/qutip/qutip/pull/1832>`_ by Eric Giguère)
+- Ensured that tidyup's default tolerance is read from settings at each call. (`#1830 <https://github.com/qutip/qutip/pull/1830>`_ by Eric Giguère)
+- Fixed ``scipy.sparse`` deprecation warnings raised by ``qutip.fast_csr_matrix``. (`#1827 <https://github.com/qutip/qutip/pull/1818>`_ by Simon Cross)
```
Comment (Ericgig) [inline doc/changelog.rst:51]:
```suggestion
- Fixed ``scipy.sparse`` deprecation warnings raised by ``qutip.fast_csr_matrix``. (`#1827 <https://github.com/qutip/qutip/pull/1827>`_ by Simon Cross)
```

**Comment by hodgestar** [comment]
> PRs that slipped in without the milestone: #1840, #1828

Thanks. Added.

**Pull request merged**